lib/cli.js

```diff
@@ -53,7 +53,9 @@ module.exports = (process) => {
       npm.config.set('usage', false, 'cli')
     }
 
-    npm.updateNotification = await updateNotifier(npm)
+    updateNotifier(npm).then(notification => {
+      npm.updateNotification = notification
+    })
```

Comment:
```suggestion
    updateNotifier(npm)
```

Now that you've done this iteration, it's more clear that the `updateNotifier` should assign this attribute itself. It's already being passed in the npm object and can do this asynchronous action on its own.